app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,12 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
```

Comment:
Fine with me, I just thought it was nicer but it's unnecessarily complex. I will figure out how to get the safe parameters from Apipie instead of `permitted_attributes` and I'll update this.